tests/complete/main.tf

```diff
@@ -149,8 +149,14 @@ module "eks_blueprints_addons" {
   enable_aws_load_balancer_controller = true
   enable_metrics_server               = true
   enable_vpa                          = true
-  enable_aws_for_fluentbit            = true
   enable_fargate_fluentbit            = true
+  enable_aws_for_fluentbit            = true
+  aws_for_fluentbit = {
```

Comment:
somewhat contrived since in order for this to actually send logs to S3, the values output would need to be updated for S3. But, it does test and validate the permission change works as intended

Ref: https://github.com/aws-ia/terraform-aws-eks-blueprints-addons/issues/118